docs/api/plugins/null.rst

```diff
@@ -0,0 +1,5 @@
+:mod:`certbot.plugins.null`
```

Comment:
I wouldn't include this file, is there a reason why you want to have api documentation for it?